### PR TITLE
fix(types): correct 'decrypt_io' param and return type annotations

### PR DESCRIPTION
**Pull request by Alchemyst0x**
The 'decrypt_io' function was incorrectly parameterized with a "recipients" 
parameter instead of "identities." Additionally, the function was typed as 
returning None, but it should return bytes.

This commit corrects both the parameterization and the return type annotations 
to align with the Rust code, which expects "identities" and returns bytes.

This may correct an issue that existed prior to my previous commit, but it 
was overlooked until now.

### CODE REVIEW

File: pyrage-stubs/pyrage/__init__.pyi [hunk 1]
```diff
@@ -37,6 +37,7 @@ class IdentityError(Exception):
 def encrypt(plaintext: bytes, recipients: Sequence[_Recipient]) -> bytes: ...
 def encrypt_file(infile: str, outfile: str, recipients: Sequence[_Recipient]) -> None: ...
 def encrypt_io(in_io: BufferedIOBase, out_io: BufferedIOBase, recipients: Sequence[_Recipient]) -> bytes: ...
+
 def decrypt(ciphertext: bytes, identities: Sequence[_Identity]) -> bytes: ...
 def decrypt_file(infile: str, outfile: str, identities: Sequence[_Identity]) -> None: ...
-def decrypt_io(in_io: BufferedIOBase, out_io: BufferedIOBase, recipient: Sequence[_Identity]) -> None: ...
+def decrypt_io(in_io: BufferedIOBase, out_io: BufferedIOBase, identities: Sequence[_Identity]) -> bytes: ...
```
Comment (woodruffw) [inline pyrage-stubs/pyrage/__init__.pyi:43]: I don't think this return type is right -- `decrypt_io` writes into `out_io`, it doesn't return anything.

Comment (woodruffw) [reply pyrage-stubs/pyrage/__init__.pyi:43]: Ref:

https://github.com/woodruffw/pyrage/blob/0d01c462b9b108ae8dd27791608e5e4c4deb0045/src/lib.rs#L293-L319

Comment (woodruffw) [inline pyrage-stubs/pyrage/__init__.pyi:43]:
```suggestion
def decrypt_io(in_io: BufferedIOBase, out_io: BufferedIOBase, identities: Sequence[_Identity]) -> None: ...
```

Comment (Alchemyst0x) [reply pyrage-stubs/pyrage/__init__.pyi]: Thanks for the clarification. Sorry for the delay, I just saw the notification for this.

**Review by woodruffw — APPROVED** [review]

**Comment by woodruffw** [comment]
Thanks for fixing this -- I've released a new patch version of the stubs package, and it should be live on PyPI shortly.

**Pull request merged**